components/list/list-item-link-mixin.js

```diff
@@ -66,13 +115,13 @@ export const ListItemLinkMixin = superclass => class extends ListItemMixin(super
 		this.shadowRoot.querySelector(`#${this._primaryActionId}`).click();
 	}
 
-	_renderPrimaryAction(labelledBy) {
+	_renderPrimaryAction(labelledBy, content) {
```

Comment:
Unsure about [this reference to _renderPrimaryAction](https://search.d2l.dev/xref/Brightspace/cm-coursemerchant-main/files/lit_components/d2l_cm_components.js?r=2bf6fb5c79eaafa35a50ed226ab47656f1123ce4#3885).

Reply:
It looks like some sort compiled version of our components (that line looks to be `list-item-mixin`). There's also the definition of `_renderPrimaryAction` in there so I think if we changed this then that would update whenever that happens for it.

Reply:
Yeah the formatting made me think that too.